freqtrade/freqtradebot.py

```diff
@@ -657,6 +665,40 @@ def handle_trade(self, trade: Trade) -> bool:
         logger.info('Found no sell signals for whitelisted currencies. Trying again..')
         return False
 
+    def handle_stoploss_on_exchage(self, trade: Trade) -> bool:
+        # Check uf trade is fulfulled in which case the stoploss
+        # on exchange should be added immediately if stoploss on exchnage
+        # is on
+        result = False
+        if trade.is_open and trade.open_order_id is None and trade.stoploss_order_id is None:
```

Comment:
can be simplified to 

` if trade.is_open and not trade.open_order_id and not trade.stoploss_order_id:`